src/Jackett.Common/Definitions/shareuniversity.yml

```diff
@@ -0,0 +1,143 @@
+---
+  site: ShareUniversity
+  name: ShareUniversity
+  description: "ShareUniversity is a Private Torrent Tracker for High Quality Nordic Content"
+  language: en-us
+  type: private
+  encoding: UTF-8
+  links:
+    - https://shareuniversity.org
+
+  caps:
+    categorymappings:
+      - {id: 1, cat: Movies}
+      - {id: 8, cat: Movies}
+      - {id: 2, cat: TV}
+      - {id: 9, cat: TV}
+      - {id: 12, cat: TV/Anime}
+      - {id: 6, cat: Other}
+      - {id: 13, cat: Audio/Audiobook}
+      - {id: 4, cat: Books/Ebook}
+      - {id: 16, cat: E-learning}
```

Comment:
category 16 is not valid. see https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt#L608-L695 for valid values